llvm/docs/DeveloperPolicy.rst

```diff
@@ -539,11 +539,13 @@ What are the expectations around a revert?
 Obtaining Commit Access
 -----------------------
 
-We grant commit access to contributors that can provide a valid justification.
-If you would like commit access, please use this `link <https://github.com/llvm/llvm-project/issues/new?title=Request%20Commit%20Access%20For%20%3Cuser%3E&body=%23%23%23%20Why%20Are%20you%20requesting%20commit%20access%20?>`_ to file
+Once you have 3 or more merged pull requests, you may use this
+link <https://github.com/llvm/llvm-project/issues/new?title=Request%20Commit%20Access%20For%20%3Cuser%3E&body=%23%23%23%20Why%20Are%20you%20requesting%20commit%20access%20?>`_ to file
 an issue and request commit access. Replace the <user> string in the title
 with your github username, and explain why you are requesting commit access in
-the issue description. If approved, a GitHub invitation will be sent to your
+the issue description.  Once the issue is created, you will need to get two
+current contributors to ack your requests before commit access will be granted.
```

Comment:
Maybe "to support" instead of "to ack"?

Reply:
Would it be too much to give a little hand here and say something like: "Providing evidence of your contributions would help reviewers support your case. This includes links to your pull requests, forum posts, etc."

Reply:
> Would it be too much to give a little hand here and say something like: "Providing evidence of your contributions would help reviewers support your case. This includes links to your pull requests, forum posts, etc."

I think that risks giving the impression that more is required than having had 3 PRs merged, and there's not consensus on that.

I do think it would be helpful to give advice like "Once the issue is created, you will need to get two
current contributors to ack your requests before commit access will be granted. It is recommended that you CC reviewers of your merged patches in order to provide this approval."

Reply:
> Would it be too much to give a little hand here and say something like: "Providing evidence of your contributions would help reviewers support your case. This includes links to your pull requests, forum posts, etc."

We have automation which posts a contribution summary when someone applies for commit access and notifies people who have reviewed or committed their patches.  See https://github.com/llvm/llvm-project/issues/127077#issuecomment-2657003375

Reply:
@asb It's a good question.  For me, the ack means 1) I know this person and 2) I believe they will adhere to our Developer Policy/Code of Conduct.

Reply:
I wonder if it would be uncontroversial to say something like:

"Reviewers of your committed patches will automatically be CCed upon creating the issue. Most commonly these reviewers will provide the necessary approval, but approvals from other LLVM committers are also acceptable. Those reviewing the application will confirm that you have indeed had three patches committed, and that based on interactions on those reviews and elsewhere in the LLVM community they have no concern about you adhering to our Developer Policy and Code of Conduct."

Reply:
> "Reviewers of your committed patches will automatically be CCed upon creating the issue. Most commonly these reviewers will provide the necessary approval, but approvals from other LLVM committers are also acceptable. Those reviewing the application are confirming that you have indeed had three patches committed, and that based on interactions on those reviews and elsewhere in the LLVM community they have no concern about you adhering to our Developer Policy and Code of Conduct."

@asb Sounds good to me, I've added this paragraph to the document.